README.md

```diff
@@ -4,7 +4,7 @@
 Official Kuzzle Javascript SDK
 ======
 
-This SDK version is compatible with Kuzzle 1.0.0-RC6 and higher
+This SDK version is compatible with Kuzzle 1.0.0-RC8 and higher
```

Comment:
As @benoitvidis said to me (and I agree): is this really necessary?
Since we won't have any real users until v1 is out?